src/imageio/imageio_rawspeed.cc

```diff
@@ -527,9 +526,8 @@ dt_imageio_retval_t dt_imageio_open_rawspeed_sraw(dt_image_t *img, RawImage r, d
         for(int i = 0; i < img->width; i++, out += 4)
         {
           for(int k = 0; k < 3; k++)
-          {
             out[k] = (float)in(j, cpp * i + k) / (float)UINT16_MAX;
-          }
+          out[3] = 0.0f;
```

Comment:
I see that you now you put it into the outer loop, not outside just below the definition of ``out``. Do you expect the address of ``out`` to be changed from the outside?

Reply:
We iterate through width and for each position we write the three colors into the 4-float-per-pixel data used later in the pipeline. The `out[3] = 0.0f` ensures the 'mask channel' to be _defined_ and never be a NaN for every pixel.

Reply:
okaaaay, now I understand, nevermind my dumb comment then :see_no_evil: